apps/channels/models.py

```diff
@@ -33,6 +35,10 @@ def save(self, *args, **kwargs):
             token = self.extra_data.get("bot_token", "")
             logging.error(f"Unable set Telegram webhook with token '{token}'")
 
+    @property
+    def channel_display_name(self):
+        return PLATFORM_DISPLAY_NAME[self.platform]
```

Comment:
Can you not just use the `choices` and the built in `get_platform_display` for this? https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model.get_FOO_display

Reply:
@czue Yeah, that was the direction what I initially took, but decided to rather decouple the display text from the internal slugs with this `PLATFORM_DISPLAY_NAME` dict. 

Granted, the `choices` display text is currently the same as their slugs, so maybe the question needs to be asked whether or not we want the display text (take for example whatsapp) to be "whatsapp" or "WhatsApp"?

I know it's a very small thing, but in my opinion rather get it right now than having to fix it later on. Thoughts?

Reply:
the second value is intended to be the display name. so i'd think we can update the choices dict to use the display names we want and then follow convention rather than re-implement the same thing in a different way.

Reply:
@czue Good point.